olivetti.el

```diff
@@ -1,9 +1,14 @@
 ;;; olivetti.el --- Minor mode for a nice writing environment -*- lexical-binding: t; -*-
 
-;; Copyright (c) 2014-2016 Paul Rankin
+;; Copyright (c) 2014-2017 Paul Rankin
 
 ;; Author: Paul Rankin <hello@paulwrankin.com>
 ;; Keywords: wp
+;; Version: 1.5.8
```

Comment:
Can we ditch the version? This is added by MELPA from the latest tag, so if I do it manually I’ll always forget and it will get out of whack.

Reply:
["There **must** [emphasis theirs] be a Version header, unless you’re only supporting MELPA"](https://www.emacswiki.org/emacs/MakingPackages#toc1)  My interest is in packaging Olivetti for Debian, where this requirement is maintained, because it's always needed for Emacs' Package.el dependency resolution.  It's cool that MELPA adds this automatically.  Honestly, I didn't think you'd be opposed to it, because fountain-mode has this header :-)  If you'd like, I'd be happy to find or write a shell script (probably used as a git tag hook), lisp function, or magit do-this-before-tagging-release-and-then-tag-release-hook.  Worst-case scenario, I'll maintain a Debian-specific patch.  P.S. I think debian ftp-masters are impressed with fountain-mode, because it made it from the NEW queue to the archive in less than two days!

Reply:
Cool. I’m on macOS so I’m unaware of how Debian packages things. If you know of some git hook script to add this automatically from the tag it would be great if you could let me know.